### DISCUSSION TITLE
Problem with conditional

### DISCUSSION THREAD

**Discussion by localghost8000** [statamic/cms#8380] [Troubleshooting] [answered]
Can anyone point me in the right direction for an issue I'm having? I assume this must be ~somewhere~ in the Antlers docs. 

I'm using the Statamic peak theme. I have a relatively simple component that has a Theme setting via a select field, which changes the colors of the component. Due to the theming, each client has a light logo and a dark logo. As an example, this is working fine.

```
    {{ clients }}
        {{ if theme == 'onyx' }}
            {{ partial:statamic-peak-tools::components/picture :image="logo_light" }}
        {{ else }}
            {{ partial:statamic-peak-tools::components/picture :image="logo_dark" }}
        {{ /if }}
    {{ /clients }} 
```
    
However, I'd really like to try to simplify this. I'm used to twig, where I would _set_ the variable. I've tried this:
    
```
    {{ clients }}
    {{ partial:statamic-peak-tools::components/picture image="{{ theme == 'onyx' ? 'logo_light' : 'logo_dark' }}" }}
    {{ /clients }}
```

Which doesn't cause errors, but non-the-less fails to return a result. Any advice on how to approach this the Antlers way, or how to debug would be appreciated.

**Comment by ryanmitchell** [comment, marked as answer]
Use single braces inside tags... 

```
    {{ clients }}
    {{ partial:statamic-peak-tools::components/picture image="{ theme == 'onyx' ? 'logo_light' : 'logo_dark' }" }}
    {{ /clients }}
```

**Comment by jackmcdade** [comment]
Bingo, @ryanmitchell. Docs on that are right here: https://statamic.dev/antlers#tag-parameters
